tests/data-generator.js

```diff
@@ -51,16 +56,16 @@ async function getKeys() {
 }
 
 /**
- * An extremely basic VP prover. This is not final
+ * An extremely basic Vp creator. This is not final
```

Comment:
Again, these are human facing abbreviations, so need to stay all caps. Maybe a global search and replace got too greedy? 😀

Reply:
```suggestion
 * An extremely basic VP creator. This is not final
```

Reply:
I was actually just being thorough anyways here ya go: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/60/commits/eda2a3b473cab756fa890aed271cab870fdf55db